### DISCUSSION TITLE
[mini.pick] how to override `cmdheight` for picker window?

### DISCUSSION THREAD

**Discussion by jamestansx** [nvim-mini/mini.nvim#1069] [Q&A] [answered]
I notice that `cmdheight` is forced to set to 1 when picker window is created, but I have a `cmdheight` of 2. How could I override cmdheight for the picker window?

**Comment by echasnovski** [comment, marked as answer]
You can do that by creating an autocommand for [`MiniPickStart`](https://github.com/echasnovski/mini.nvim/blob/db52ea9a1fd5183bb15f557db3fd6e504099bf06/doc/mini-pick.txt#L140) `User` event. Like this:

```lua
local set_cmdheight = function() vim.o.cmdheight = 2 end
vim.api.nvim_create_autocmd('User', { pattern = 'MiniPickStart', callback = set_cmdheight })
```

I thought about altering command line height only if it is zero, but always setting it to 1 seems like a good design because it enforces the biggest  vertical space it can get.

**Reply by jamestansx** [reply]
Thanks for the quick answer!

> but always setting it to 1 seems like a good design because it enforces the biggest vertical space it can get.

Agreed. But I prefer my statusline position to be static, I don't mind losing one column. I think it's very rare that one might set their cmdheight > 3 (right?)

**Reply by echasnovski** [reply]
> I think it's very rare that one might set their cmdheight > 3 (right?)

Sure. I'd even go as far as to say that it is not that common to have command height more than 1 :) To be exact, it is 11% of users, [based on the survey](https://gist.github.com/echasnovski/fa70dc75c475369747d2a485a13303fb#file-neovim-survey-summary_02_all-csv). More than I'd guess, though.
